dashboard/test/models/user_test.rb

```diff
@@ -3802,7 +3802,7 @@ def hide_scripts_in_sections(section1, section2)
     test 'can get next_unpassed_visible_progression_level, last level complete, but script not complete, first hidden' do
       student = create :student
       teacher = create :teacher
-      script = create(:script, :with_levels, levels_count: 3)
+      script = create(:script, :with_levels, lessons_count: 3, levels_count: 1)
```

Comment:
needed because the original implementation created each level in its own lesson.